composer.json

```diff
@@ -32,6 +32,9 @@
         "psr/log": "^1.0",
         "symfony/phpunit-bridge": "^4.0"
     },
+    "provide": {
+        "php-http/client-implementation": "1.0"
```

Comment:
that's actually wrong, as your clients don't implement the HTTPlug interface, but only the PSR one (and so they are not php-http client implementations)

Reply:
I think you are mistaken. We do implement the HTTPluginterface

https://github.com/kriswallsmith/Buzz/blob/v0.17.0/lib/Client/BuzzClientInterface.php

Reply:
hmm, I indeed missed it.